src/operations/booleans/mayOverlap.test.js

```diff
@@ -0,0 +1,70 @@
+const test = require('ava')
+
+const {EPS} = require('../../core/constants')
+
+const {geom3} = require('../../geometry')
+
+const {cuboid} = require('../../primitives')
+
+const mayOverlap = require('./mayOverlap')
+
+test.only('mayOverlap: determination of overlap is perfect', t => {
```

Comment:
is perfect ? perhaps something like reliable"  is a better wording?

Reply:
done